src/main/java/org/zalando/pazuzu/feature/FeatureMetaDto.java

```diff
@@ -0,0 +1,107 @@
+package org.zalando.pazuzu.feature;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FeatureMetaDto {
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("description")
+    private String description;
+    @JsonProperty("author")
+    private String author;
+    @JsonProperty("updated_at")
+    private ZonedDateTime updatedAt;
+    @JsonProperty("dependencies")
+    private List<String> dependencies = new ArrayList<>();
+
+    public static FeatureMetaDto of(Feature feature) {
+        return new FeatureMetaDto().setName(feature.getName())
+                .setDescription(feature.getDescription())
+                .setDependencies(feature.getDependencies().stream().map(Feature::getName).collect(Collectors.toList()));
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public FeatureMetaDto setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public FeatureMetaDto setDescription(String description) {
+        this.description = description;
+        return this;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public FeatureMetaDto setAuthor(String author) {
+        this.author = author;
+        return this;
+    }
+
+    public ZonedDateTime getUpdatedAt() {
+        return updatedAt;
+    }
+
+    public FeatureMetaDto setUpdatedAt(ZonedDateTime updatedAt) {
+        this.updatedAt = updatedAt;
+        return this;
+    }
+
+    public List<String> getDependencies() {
+        return dependencies;
+    }
+
+    public FeatureMetaDto setDependencies(List<String> dependencies) {
```

Comment:
:+1:
I actually like the switch from dependencies being strings and not objects. 
That keeps the features loosely interdependent and makes meta lighter.